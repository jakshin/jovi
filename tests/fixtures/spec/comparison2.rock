If "1" is 1 Say "1 is 1"
If "2" ain't Mysterious Say "2 ain't mysterious"
If "02" is lower than "10" Say "02 is lower than 10"
If True is lower than 10 Say "True < 10" (nope, it isn't)
