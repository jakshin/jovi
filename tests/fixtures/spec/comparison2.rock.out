1 is 1
2 ain't mysterious
02 is lower than 10
